airflow/models/dag.py

```diff
@@ -694,22 +701,27 @@ def has_dag_runs(self, session=None, include_externally_triggered=True) -> bool:
 
     @property
     def dag_id(self) -> str:
+        """Get the dag_id"""
         return self._dag_id
 
     @dag_id.setter
     def dag_id(self, value: str) -> None:
+        """Set dag_id"""
         self._dag_id = value
 
     @property
     def full_filepath(self) -> str:
+        """Dag location"""
         return self._full_filepath
 
     @full_filepath.setter
     def full_filepath(self, value) -> None:
+        """update full filepath"""
         self._full_filepath = value
 
     @property
     def concurrency(self) -> int:
+        """Max active tasks limit of a dag"""
```

Comment:
I find these type of DAG comments _less_ helpful than no doc at all.

I'd rather we ignored the warning on these ones.

Reply:
I have removed less helpful doc strings including this one, thanks for your advice:)

Reply:
@ashb Could you help to have a look again?